packages/fxa-metrics-processor/src/lib/amplitude/index.ts

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  Services,
+  PlainEvents,
+  FuzzyEvents,
+  RawEvent,
+  EventContext,
+  AmplitudeHttpEvent,
+} from './transforms/types';
+import { createServiceNameAndClientIdMapper } from './transforms/common';
+import { createAmplitudeEventTransformer } from './transforms/amplitude-event';
+import {
+  createAmplitudeIdentityEvent,
+  AmplitudeIdentifyEvent,
+} from './transforms/amplitude-identify';
+
+export type AmplitudeRequestPayloads = {
+  http: AmplitudeHttpEvent;
+  identify?: AmplitudeIdentifyEvent;
+};
+
+/**
+ * Create a transform function from the given mappings.  A transform function
+ * should be created per service (specifically for each of the "eventSource"
+ * values in RawEvent).
+ *
+ * The transform function returns an object keyed by the Amplitude API for
+ * which the value is intended.  There are two possible keys: "http" and
+ * "identify".
+ *
+ * The "identify" property is optional.  However, if it does exist, then the
+ * API request for it MUST be ahead of the request for the http API.
+ *
+ * @param services An object of client-id:service-name mappings
+ * @param plainEvents  An object of name:definition event mappings, where
+ *                     each definition value is itself an object with `group`
+ *                     and `event` string properties.
+ *
+ * @param fuzzyEvents A map of regex:definition event mappings. Each regex
```

Comment:
Not related to this PR (since it was already here), but fuzzy events have always been a bit fuzzy to me. After reading the description, I still have a hard time understanding what this property does. 

We could probably link to some docs to provide context on the amplitude taxonomy and what different terms mean.